src/main/assembly/dist/cfg/config.yml

```diff
@@ -11,12 +11,25 @@ server:
     - type: http
       port: 20341
 
-dataverseDatabase:
+database:
```

Comment:
Why did you rename this? This service only works with Dataverse, hence`dataverseDatabase`.

Reply:
I thought you made a mistake, but I renamed it to match your original intentions again